README.md

```diff
@@ -96,86 +87,48 @@ $ gem install coverband
 
 # Configuration
 
-After you have the gem, you will want to setup the needed integration:
+After you have the gem, you may need to configure additional options:
 
-1. Coverband Config File Setup
-2. Configure Rake
-3. Insert Coverband Rack Middleware
+### 1. Verify Rake
 
-### 1. Coverband Config File Setup
+After installing the gem in Rails you should have these Rake tasks
```

Comment:
I think it's important that the readme gives a feeling of a drop in gem.. especially for rails. Was thinking the basic installation would be communicated near the top. Those who are interested in additional configuration options have to do a bit more digging in. Something like:

## General installation
### Rails
### Sinatra
## Report Generation
### Rake Task
### Viewing the Report in App
#### Rails
#### Sinatra
## Additional Configuration

Reply:
Just wanted to add.. happy to take a crack at it if you like the basic idea.

Reply:
Thanks, I took another pass moving some of the more advanced stuff to the end and trying to simplify the earlier parts of the Readme. Take a look and let me know if that looks better @kbaum